### PR TITLE
feat: add teamcity

### PR DESCRIPTION
**Pull request by azz**
Adds support for TeamCity.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,13 +80,7 @@
   },
   "xo": {
     "extends": [
-      "prettier"
```
Comment (azz) [inline package.json]: Simplified this config a bit, it's exactly the same: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

**Comment by pvdlg** [comment]
Looks great! Thanks!

Could you just add `teamcity` to the `package.json` `keywords`? Thanks

File: index.js [hunk 2]
```diff
@@ -3,18 +3,19 @@
 const git = require('./lib/git');
 
 const services = {
-  travis: require('./lib/travis'),
```
Comment (azz) [inline index.js]: Had a merge conflict here so I sorted this, too.

Comment (pvdlg) [reply index.js]: 👍

**Pull request merged**